app/models/application_record.rb

```diff
@@ -137,4 +147,29 @@ def self.ethnicity_eeo_data(records)
       [ethnicity, specific_records.count, specific_records.interview_count]
     end
   end
+
+  def self.notification_emails(records)
+    emails = Hash.new { |h, k| h[k] = {} }
```

Comment:
How about the word `fill` in favor of `build` since that is what the method is really doing.